### DISCUSSION TITLE
Default logo

### DISCUSSION THREAD

**Discussion by tbertels** [mpc-qt/mpc-qt#250] [General]
While the screenshot at https://mpc-qt.github.io/ uses the "[Triangle in circle](https://github.com/mpc-qt/mpc-qt/blob/master/images/logo/triangle-circle.svg)" logo, the current default one is "[Color film stock](https://github.com/mpc-qt/mpc-qt/blob/master/images/logo/film-color.svg)".

Shouldn't either the screenshot or the default logo be changed?
And what should it be?

![Copie d'écran_20241108_114856](https://github.com/user-attachments/assets/9bbfbae4-168c-4f96-9de1-1492875b5aa0)

**Comment by cmdrkotori** [comment]
You're right, everything should be consistent.

I did that because I thought "triangle in circle" looked better aesthetically in a screenshot, even if it it's a bit generic for a media player.  The only reason why "color film stock" has stayed the default is because it looks like what the icon is.

I think in an ideal world, everything would match: the logo, the icon, the screen shot.

So I think our paths forward are to change everything to "triangle in circle", or to update the screenshots with "color film stock".  I think the less disruptive thing to do is to update the screenshots, because some icon themes have their own version of the smiley icon.

Speaking of, I've been mucking around in inkscape and came up with the following new logo. What do you think?
![diamond](https://github.com/user-attachments/assets/0f23168e-16d4-46b1-9ef1-782736230183)

**Reply by tbertels** [reply]
Yes, if we could keep the smiley, it's kinda fitting with the Qute part.

About the new one, I think it looks a bit generic and we'd loose the Qute identity.
The reflect looks great though.

Was it you who made the original "color film stock" one?
Perhaps we could improve on it.
Maybe that's just me but it seems to look rather greenish right now.
Why not remove the perforations as the future is numeric.

Here's a variation based on "Color film stock" (the reflection is wrong):

The in-app logo (vs icon) could be a cinema screen in a cinema auditorium as mpc-qt can be/is used for home cinema.

![film-blank](https://github.com/user-attachments/assets/eb1b3ea5-0386-4d63-a131-c027325af778)

**Reply by tbertels** [reply]
By the way, you didn't comment on my slight change of [Yasand123](https://github.com/Yasand123)'s icon: https://github.com/mpc-qt/mpc-qt/issues/111#issuecomment-2306729330
I take that as meaning that I'm not ready to become a graphic designer. 😅

**Reply by cmdrkotori** [reply]
>The in-app logo (vs icon) could be a cinema screen in a cinema auditorium as mpc-qt can be/is used for home cinema.

Yes, it looks better without the film.  I think if we do this, we remove the black-and-white film stock, as it is practically the same as this.  And we rename "color film stock" to "cinema screen" or some such.

>the reflection is wrong

The reflection is on "Layer 2", it's a copy of "Layer 1" but flipped.  You can also get to the objects there by holding down alt+ctrl.  The "reflection" is created by drawing a fade to black in "Layer 3" over the top of it.  Turn the visibility of "Layer 3" off and you'll see what's going on.


>I take that as meaning that I'm not ready to become a graphic designer. 😅

Ah sorry about that. The thing is, I'm not ready to become a graphic designer too!  If I'm stumped at something I tend to leave the thread there and hope to come back to after some inspiration strikes.  I'll have another look at it now.

**Reply by tbertels** [reply]
> The reflection is on "Layer 2", it's a copy of "Layer 1" but flipped. You can also get to the objects there by holding down alt+ctrl. The "reflection" is created by drawing a fade to black in "Layer 3" over the top of it. Turn the visibility of "Layer 3" off and you'll see what's going on.

Thanks, here it is with fixed reflection:

![cinema-screen](https://github.com/user-attachments/assets/0dafa8ca-8e41-4b43-942c-75c43febd6a2)

If it looks good to you, I'll make a pull request with the changes you suggested.

**Reply by cmdrkotori** [reply]
LGTM, ship it.

**Reply by RomanSh-shell** [reply]
Let's keep this logo for a winamp skin)
